clang-format-diff.sh

```diff
@@ -3,6 +3,7 @@
 
 # For testing
 git br -v
```

Comment:
I think `br` is your local alias for `branch` so it won't work here.

Reply:
Oh right omg